src/armature/Node.ts

```diff
@@ -137,6 +137,100 @@ export class Node {
         return this;
     }
 
+    /**
+     * Scales the node by the specified amount, either about the node's origin or a single
+     * constrained point.
+     *
+     * @param {number | coord} amount The amount to scale by, either the same in each axis or the
+     * components for each axis.
+     * @returns {Node} The current node, for method chaining.
+     */
+    public scale(amount: number | coord): Node {
+        const amountVec = isNumber(amount)
+            ? vec3.fromValues(amount, amount, amount)
+            : Mapper.coordToVector(amount);
+        const constrainedPoints: vec3[] = [...this.held];
+
+        // If the node is attached to a parent node with an anchor, add it to the list of
+        // constrained points.
+        if (this.anchor !== null) {
+            constrainedPoints.push(this.anchor);
+        }
+
+        if (constrainedPoints.length > 1) {
+            throw new Error("Can't scale when more than one point is held!");
+        }
+
+        const anchor =
```

Comment:
Huge nit, but could you please style this ternary assignment like the above one?

Reply:
autoformatter just resets it :'(

Reply:
ahh, gotchu, then :shipit: